modules/learn/pages/security/certificates.adoc

```diff
@@ -233,6 +236,20 @@ For example, a delimiter of `.` causes the substring `.couchbase.com` to be disc
 
 For step-by-step instructions, see xref:manage:manage-security/enable-client-certificate-handling.adoc[Enable Client Certificate Handling].
 
+[#deprecation-of-subject-common-name]
+== Deprecation of Subject Common Name
+
+Use of the _Subject Common Name_ to identify either a server or a client is now deprecated (see https://tools.ietf.org/html/rfc6125#section-6.4.4[section 6.4.4 of RFC 6125^]).
+For Couchbase Server, this means that:
+
+* The node-certificate for each server in the cluster must specify at least one _Subject Alternative Name_, in order to be identified: typically, the node's IP address or DNS name should be provided in this way.
```

Comment:
I would not use word "must". I would say "We recommend", as we are not removing the support for CN.

Reply:
Ok, I've changed accordingly.